.github/workflows/integration.yaml

```
@@ -3,45 +3,27 @@ permissions:
   contents: read
 jobs:
   nix:
-    if:
-      ${{ github.event_name == 'workflow_call' || github.event_name ==
-      'workflow_dispatch' || github.event.pull_request.draft == false }}
+    if: |
```

actionlint (1.7.12)

[error] if: condition "${{ github.event_name == 'workflow_call' ||\n     github.event_name == 'workflow_dispatch' ||\n     github.event.pull_request.draft == false }}\n" is always evaluated to true because extra characters are around ${{ }}

(if-cond)